proto/zitadel/user/schema/v3alpha/user_schema.proto

```diff
@@ -79,6 +79,8 @@ message SearchQuery {
     TypeQuery type_query = 5;
     // Limit the result to a specific state of the schema.
     StateQuery state_query = 6;
+    // Limit the result to a specific schema ID.
+    IDQuery id_query = 7;
```

Comment:
since the query can also be used in a subquery, you could also use it to get a list of some ids or to exclude a specific id (e.g. get active schema exept a specific one)